### PR TITLE
add LICENSE

### PR DESCRIPTION
**Pull request by cazemar**

### CODE REVIEW

**Review by Niko-r — APPROVED** [review]

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
+                    The Content Square iOS SDK License 1.0
```
Comment (mathieu-ouvrard) [inline LICENSE:2]: Contentsquare

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+                        (c) Content Square SAS 2023
```
Comment (mathieu-ouvrard) [inline LICENSE:3]: Contentsquare

File: LICENSE (hunk 1 above)
Comment (cazemar) [reply LICENSE:2]: legal name, not the commercial name.

File: LICENSE (hunk 2 above)
Comment (cazemar) [reply LICENSE:3]: legal name, not the commercial name.

**Pull request merged**